network_analysis/closest-facility-static/README.md

```diff
@@ -1,36 +1,30 @@
-# Closest Facility (Static)
+# Closest facility (static)
 
 Find routes from several locations to the respective closest facility.
 
-![](ClosestFacilityStatic.png)
+![Image of find closest facility static](ClosestFacilityStatic.png)
 
 ## Use case
 
-Quickly and accurately determining the most efficient route between a location and a facility is a frequently encountered task (e.g. emergency services).
+Quickly and accurately determining the most efficient route between a location and a facility is a frequently encountered task. For example, a city's fire department may need to know which firestations in the vicinity offer the quickest routes to multiple fires. Solving for the closest fire station to the fire's location using an impedance of "travel time" would provide this information.
 
 ## How to use the sample
 
-Click the 'Solve Routes' button to determine and display the route from each incident (fire) to the nearest facility (fire station).
+Click the 'Solve Routes' button to solve and display the route from each incident (fire) to the nearest facility (fire station).
 
 ## How it works
 
-To display a `ClosestFacilityRoute` between several incidents and facilities:
-
-1. Create a `ClosestFacilityTask` using a Url from an online service.
+1. Create a `ClosestFacilityTask` using a URL from an online service.
 2. Get the default set of `ClosestFacilityParameters` from the task: `closestFacilityTask.createDefaultParametersAsync().get()`.
-3. Build a list of all Facilities and Incidents:
-    * Create a `FeatureTable` using `ServiceFeatureTable(Uri)`.
-    * Query the `FeatureTable` for all `Feature`s using `.queryFeaturesAsync(queryParameters)`.
-    * Iterate over the result and add each `Feature` to a `List`, instantiating the feature as a `Facility` or `Incident`.
+3. Build a list of all `Facilities` and `Incidents`:
```

Comment:
adjusting this to reflect the API class name even though grammatically it looks wrong!
```suggestion
3. Build a list of all `Facility`s and `Incident`s:
```

Reply:
Yeah the parsed MD looks a bit weird, but it's definitely better the way you suggest.